scipy/optimize/tests/test_linprog.py

```diff
@@ -661,6 +661,24 @@ def cb(xk, **kwargs):
 class BaseTestLinprogIP(LinprogCommonTests):
     method = "interior-point"
 
+    def test_bounds_equal_but_infeasible(self):
+        c = [-4, 1]
+        A_ub = [[7, -2], [0, 1], [2, -2]]
+        b_ub = [14, 0, 3]
+        bounds = [(2, 2), (0, None)]
+        res = linprog(c=c, A_ub=A_ub, b_ub=b_ub, bounds=bounds,
+                      method=self.method)
+        _assert_infeasible(res)
+
+    def test_bounds_equal_but_infeasible2(self):
```

Comment:
There's probably some potential to reduce code duplication via a parametrized pytest approach since the input data is the same for both tests with the exception of the argument specifications (i.e., `A_eq` vs. `A_ub`). I'm not sure it would be that much cleaner though since you'd likely need to specify the default `None` argument for i.e., `A_eq` when using `A_ub`, but just a thought given the test similarities.

A pytest fixture might be another option or something that could be used in combination so the parametrization isn't too bulky.